allennlp/tests/semparse/domain_languages/wikitables_language_test.py

```diff
@@ -0,0 +1,696 @@
+# pylint: disable=no-self-use,invalid-name,too-many-public-methods
```

Comment:
Looks like these tests are a union of the executor tests and world tests with the types updated. I skimmed through them. Please let me know if there are any news tests, and I'll look at them closely.

Reply:
I made some minor changes to some of them (mostly updating the types), and removed a few that were no longer necessary, but didn't add any more.